internal/cmd/flink/command_statement_describe.go

```diff
@@ -16,7 +16,6 @@ func (c *command) newStatementGetDetailsCommand() *cobra.Command {
 		RunE:              c.statementDescribe,
 	}
 
-	c.addComputePoolFlag(cmd)
```

Comment:
What if we want to describe a statement in a different compute pool than the one that's currently selected?

Reply:
It's usually best to use `confluent flink compute-pool use`, but when you need to do a one-off operation, using `--compute-pool` is more convenient, right?

Reply:
This would diverge from how our other commands work:
* `confluent environment use` has a corresponding `--environment` flag
* `confluent kafka cluster use` has a corresponding `--cluster` flag
* `confluent context use` has a corresponding `--context` flag
* `confluent api-key use` has a corresponding `--api-key` flag
* etc.

Reply:
Statements don't belong to compute pools. They belong to `env`, `region` and `cloud`. Updated the CLI to reflect this

Reply:
> Statements don't belong to compute pools. They belong to env, region and cloud.

Should we also remove `--compute-pool` from `flink statement list`, then?

Reply:
No, `compute-pool` in `statement list` is an optional filter